### ISSUE TITLE
Feature Request: Server Status

### ISSUE THREAD

**Issue by alloin** [AHAAAAAAA/PokemonGo-Map#2321]
Server Status for chosen login method.

Basically a "server down" or "server up" text or button in the top right showing the server status.
If the user uses Ptc, it checks and returns the Ptc login status, same for Google.

If this is too much work, could anyone suggest a way of doing this?
Maybe scrape the http://ispokemongodownornot.com/ details.

Thanks

**Comment by gigaguy** [comment]
Another option here would be to fail back and forth between a google and PTC account.  If PTC is down, use google, vice versa.

**Comment by trippyhat** [comment]
I don't think we need to scrape for details since server outage may depend on the player's geolocation. It might be easier if we do this:

If unable to log into google/PTC after X amount of tries, then we declare it as 'server is down' and retry again in Y minutes, where Y is like every half hour.

**Comment by alloin** [comment]
> If unable to log into google/PTC after X amount of tries, then we declare it as 'server is down' and retry again in Y minutes, where Y is like every half hour.
Yea that would be the easiest solution I think

**Comment by andrewda** [comment]
I agree with @trippyhat. Scraping is controversial and is terribly messy, so doing it that way is probably best.
